### ISSUE TITLE
There’s No Place Like Home - Sunday 3pm

### ISSUE THREAD

**Issue by eximious** [CLSummit/CLS2018#13]
There’s No Place Like Home: How Do We Provide Organizational Homes to FLOSS Communities? – Bradley Kuhn, Software Freedom Conservancy

FLOSS initiatives usually need an organizational home. There are many trade-offs in deciding how to chose or create one, and projects’ needs differ greatly. Let’s explore what organizational homes should do in this session.

Why charities?
- Mostly worked in US
- Transparency with IRS

They have a legal obligation to act in the public interest. 
- Rules against misappropriation of 501c(3) funds

Two other forms of orgs that end up acting as fiscal homes

#1: 
    501c(6): Trade associations: Act in common business interests of members
    Collect dues to get $
e.g. Chamber of Commerce, Portland Business Alliance

#2:
    For-profit company

Lots of different upsides and downsides

1901 association in France
- French tax benefit is credit, instead of deduction, so get lots of funding
- Rules for how they spend that $, but they must spend the $ France

Germany - EV
More like a club or 501(c)7
More like a 501c(4) (these are lobbying orgs)
Strict rules
- Document foundation (home of LibreOffice)

SW in the Public Interest
Associated with Debian
Very lightweight: Donations & travel expenses

Different sponsors have different rules

Different sponsors have different levels of support

B-Corp
e.g. All Things Open
Some component is for betterment of world

Different projects belong in different homes. Good to have many options. Not just company, trade association.
Concern about organization proliferation & capacity problems
Conservancy often being imitated

Q: In a model where you have an umbrella org managing many projects, how are the finances between projects compartmentalized?
- Conservancy has an earmarked fund for salary ~10%. 90% spent based on council, unless it doesn't align with org mission
- Apache goes into general fund & projects apply
- LF is fee for service. Project pays LF.

Q: Governance related to IP for project or project itself (change incoproration)
- Conservancy has rule where charity can't just buy stuff for 1 person. Try to model governance or need to mentor on how to do governance.
- Charity is a funny way to get a govt grant
- Apache has a specific governance model for their projects. One size fits all.

    - Easy to start with Apache

    - Bit harder to make it align with the culture. Easier with Conservancy

- Linux Foundation: Depends on who the companies giving the $.  This is based on business interest.
Services that each project derives from LF: 
https://www.linuxfoundation.org/projects/hosting/

Q: Friend just set up 501(c)3 for his project, but maybe he could have done different.  What are the tradeoffs?

Q: Can a charity have engineering team on staff, writing SW?
Goal has to be in public interest
e.g. Wikimedia
Mozilla is a more complicated example, not good for discussing in this session
Tor (The Onion Router) accepts gov't contracts for development
Ethereum Foundation pays developers

Q: Can a charity do govt contracts?
Tor.org is 501(c)3
There is rule for max overhead

Q: Can LF projects use funds not under LF?
They do
LF leans on common resource across projects (like business)

    CloudFoundry takes advantage & pays for certain

    LF doesn't hire engineers to work on CloudFoundry. The project itself decides.

LF core infrastructure initiative contracts out to individual devs out of the $ collected by members of project
Greg & Linus are employed of LF

Q: Does LF document guidance publicly?

501(c) status orgs must file 990 form >$100K or principal decision maker salary, and/or(?) top 5 contractors
Helps you figure out how much a non-profit might pay you if you want to work there
You can find many on Charity Navigator
Must provide on request for last 3 years
IRS has a CD set you can order of the 990s
FSF & Mozilla Foundation post theirs

From the IRS:
https://www.irs.gov/charities-non-profits/form-990-part-vii-and-schedule-j-reporting-executive-compensation-individuals-included

A GitLab project
Goal: Put for any OSS-related org that they came across

Is an org obliged to provide 990 beyond 3 years? No

Finding a home:
    - What do you want for your project?  Look at what services you need for your project & how much it will cost you
    - Having a home can benefit an org of even 5-6 ppl

        - Can access certain things via home (e.g. test phones)

    - Can address mailing list concerns

    - Putting $ into an individual's bank account is a terrible idea ("co-mingling") - 50% of projects that end up at Conservancy
    
    
FLOSS Foundations
https://flossfoundations.org/ 
- Mailing list for different non-profits

TLDR; Legal -> Boils down licenses to digestable level.
https://tldrlegal.com/
Choose a license.org
Choose a foundation.org -> The decision is probably more nuanced
http://chooseafoundation.com/help

How could we make this info digestable?
- Put it where ppl already are: 
    GitLab or Github mini site with a topic on when you should consider setting up an org
- Literature that projects could share with their models
- Case studies

When do you pursue opportunity for clients?
- Do you really want to do all the things as 1 person that a 5 person business would have separate roles for?
- What are the goals for your project?

Michael tried to catalog known sponsors/homes

How do you move from business to non-profit?
- Have to talk to a lawyer & an accountant, so that they can assess your situation
- e.g. Open House
- Easier to go from business to non-proft
- Charity has to be careful about how remaining assets are spent -- 501(c)3 "black hole"

JQuery decided they didn't want to operate as a charity
- (c)3 winds down involvement, then (c)6 picks up
- Found a solution where Conservancy let them set up a 501(c)6 and then sold them
- Remaining $ supported javascript in charitable way
- Stopped accepting donations for JQuery

Rival projects under same umbrella?
- Mercurial, Git, Darcs are all under Conservancy. Seems to be going fine.
- They don't get to say "help us & not them"
- Apache Foundation explicitly states they support competing projects b/c their mission is to be a neutral ground

We don't want to create a competitive market between foundations. Not productive.
e.g. Universities spend a lot on pretty buildings
Conservancy doesn't give office space

Moving between different fiscal sponsors
Better if project can figure it out up front, but project needs to find their right home.
